presto-native-execution/presto_cpp/presto_protocol/connector/hive/presto_protocol_hive.yml

```diff
@@ -0,0 +1,188 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+EnumMap:
+  AggregationNode:
```

Comment:
Are all these definitions needed? (e.g.: AggregationNode, RemoteTransactionHandle, PlanNodeId, ...)

Reply:
The python script was assuming these as mandatory so it needed a fix on the python code. I made those changes and removed these unrelated ones from all the yaml files. 

There is a corner case where, in the absence of a RowExpression defined in the yaml file, the type was not getting converted to std::shared_ptr<RowExpression> in ConnectorTableLayoutHandle implementations. I defined a new config key in the yaml file called UpdateFields, similar to the existing ExtraFields node, to resolve this issue. This approach avoids the need to create special files for HiveTableLayoutHandl,  TpchTableLayoutHandle etc